code/modules/mob/living/simple_animal/hostile/mimic.dm

```diff
@@ -119,6 +119,8 @@ var/global/list/protected_objects = list(/obj/structure/table, /obj/structure/ca
 
 /mob/living/simple_animal/hostile/mimic/copy/Life()
 	..()
+	if(!target && !client) //Objects eventually revert to normal if no one is around to terrorize

```

Comment:
You should have it checking key or ckey, as getting disconnected either doesn't, or shouldn't cause it to start attacking again.